.github/actions/clp-core-build/action.yaml

```diff
@@ -11,27 +11,25 @@ runs:
   using: "composite"
   steps:
 
+
     - name: Download submodules
       run: ./tools/scripts/deps-download/download-all.sh
-      working-directory: $GITHUB_WORKSPACE/components/core
+      working-directory: ./components/core
       shell: bash
 
     - name: Configure CMake
       # Configure CMake in a 'build' subdirectory. `CMAKE_BUILD_TYPE` is only required if you are using a single-configuration generator such as make.
       # See https://cmake.org/cmake/help/latest/variable/CMAKE_BUILD_TYPE.html?highlight=cmake_build_type
       run: cmake -B $GITHUB_WORKSPACE/build -DCMAKE_BUILD_TYPE=${{inputs.build_type}}
-      working-directory: $GITHUB_WORKSPACE/components/core
+      working-directory: ./components/core
       shell: bash
 
     - name: Build
       # Build your program with the given configuration
       run: cmake --build $GITHUB_WORKSPACE/build --config ${{inputs.build_type}}
-      working-directory: $GITHUB_WORKSPACE/components/core
-      shell: bash
-
-    - run: mkdir -p $GITHUB_WORKSPACE/build && touch $GITHUB_WORKSPACE/build/clp && touch $GITHUB_WORKSPACE/build/clg && touch $GITHUB_WORKSPACE/build/clo && touch $GITHUB_WORKSPACE/build/make-dictionaries-readable
+      working-directory: ./components/core
       shell: bash
-    
+      
```

Comment:
* Can we use hyphens in the artifact name below? It's a tiny bit easier to type, lol.
* Any disadvantages to using a multiline command instead of concatenating them with `&&`?

Reply:
Mainly did that while developing to make sure it is able to find the directory before running the tar command, but using a `;` isn't an issue

Reply:
Oh, I meant something like:

```
run: |
  cd $GITHUB_WORKSPACE/build
  tar -cf $GITHUB_WORKSPACE/clp-binaries-focal.tar ./clp ./clg ./clo ./make-dictionaries-readable
```

Reply:
Ohh. Didnt know that syntax existed. Thats cleaner, I'll do that